prometheus-exporters/ucs-exporter/templates/ucs-exporter-deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
              - {{ required ".Values.ucs.user missing" .Values.ucs.user }}
              - -p
              - {{ required ".Values.global.exporter_master_password missing" .Values.global.exporter_master_password }}
+             - -s
+             - {{ .Values.ucs.internal_cache_scrape_interval }}
```

Comment:
Please add this mapping in `values.yaml` with a default value for "normal" regions.

Reply:
What happens if not defined in secrets?
How about writing therefor
{{ if .Values.ucs.internal_cache_scrape_interval }}{{ .Values.ucs.internal_cache_scrape_interval }}{{ else }}300{{ end }}

Reply:
This one is not specified as "required", so exporter will go with default value(300), I don't think loop is required to specify this. 
@viennaa please confirm.

Reply:
The whole setting starting with `-s` need to be in the if else block. Default is 300, so nothing need to be specified indeed: https://github.com/sapcc/ucs-exporter/blob/master/ucs-exporter.py#L51-L52

So I'd go with 

```
{{ if .Values.ucs.internal_cache_scrape_interval }}
- -s
- {{ .Values.ucs.internal_cache_scrape_interval }}
{{ end }}
```

See comment in values.yaml.